docs/content/core/volumes.md

```diff
@@ -5,228 +5,179 @@ aliases:
-| ANY     | ANY             | none    | *No*      | `none` option is like `unison`, but without the auto-sync. Useful for huge projects where FS performance is a bottleneck, but when `unison` does not work for you. <br> **Pros:** maximum `cli` filesystem performance and no wait for the initial sync. <br> **Cons:** you have to copy files manually or checkout and edit files inside `cli` container only. |
-
-## Full explanation on volumes in Docksal
-
-Let's go over volumes in Docker first. 
-
-In Docker, [volumes](https://docs.docker.com/engine/admin/volumes/volumes/) are used primarily for persisting data 
-generated by and used by Docker containers. Docker automatically creates and manages volumes, storing them in a special
-location within the host machine's filesystem. There are also different volume plugins, which add support for 
-various other data storage backends for volumes.
-
-Mounting an existing folder from the host machine can also be done via volumes using 
-[bind mounts](https://docs.docker.com/engine/admin/volumes/bind-mounts/). 
-
-By default, Docksal uses the bind mount approach. 
-
-The VM layer used on macOS/Windows (through VirtualBox or Docker for Mac/Windows) adds some complexity to that, however
-that's not something you normally have to worry about. Both Docksal and Docker for Mac/Windows handle that automatically.
- 
-From the perspective of a container, a local Linux path is mounted regardless of the underlying host OS. 
-On Mac, the host filesystem is mounted with NFS, on Windows - using SMB.
-
-Let's take a look at an example.
-
-The host machine is a macOS and the codebase root directory (the "Projects" folder) is `/Users/username/Projects`. 
-This directory is mounted with the same path inside the VM: `/Users/username/Projects`. Any path within that directory 
-is exactly the same on the host and inside the VM.
-
-When a project stack is started, the project root directory (e.g., `/Users/username/Projects/myproject`) is bind mounted 
-into `/var/www` inside the containers. A corresponding line in `docksal.yml` for this would be:
+If you are familiar with Docker Compose, then you must have attached files or dirs like this:
 
 ```yaml
-version: "2.1"
-
-services:
   cli:
     volumes:
-      - ${PROJECT_ROOT}:/var/www:rw
+      - /Users/alex/mysite:/var/www:rw
 ```
 
-`${PROJECT_ROOT}` is automatically set to the project's root directory on the host.
-
-The whole mount chain looks like this (drop the last part for Linux hosts).
+That is a called a bind mount - mounting a files/folder on the host directly into a container. 
```

Comment:
```suggestion
That is a called a bind mount - mounting a file/folder directly into a container. 
```

Let us avoid confusing user about what "host" is in terms of this documentation. You and I understand there are several "hosts", but to make the whole read clear to user in this doc lets call "host" a computer that runs Docker or VM. In this case it is not correct that bind mount example above would mount files from host, as on macOS, Windows it is not from host.